### PR TITLE
Mieubrisse/param endpoint

### PR DESCRIPTION
**Pull request by mieubrisse**
NOTE: review & merge https://github.com/kurtosis-tech/ava-e2e-tests/pull/21 first!!

### CODE REVIEW

**Comment by mieubrisse** [comment]
Closing for now; will reopen later

File: gecko_client/pchain_api.go [hunk 1]
```diff
@@ -13,25 +13,25 @@ type PChainApi struct {
...
+	var response GetBlockchainStatusResponse
+	if err := json.Unmarshal(responseBodyBytes, &response); err != nil {
+		return BlockchainStatus{}, stacktrace.Propagate(err, "Error unmarshalling JSON response")
+	}
+	return response.Result, nil
 }
 
 func (api PChainApi) GetCurrentValidators() ([]Validator, error) {
-	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(pchainEndpoint, "platform.getCurrentValidators")
+	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(pchainEndpoint, "platform.getCurrentValidators", make(map[string]interface{}))
```
Comment (galenmarchetti) [inline gecko_client/pchain_api.go:34]: I think we need to make this change in the admin API as well, travisCI failing with:

The command "eval go get -v -t ./... " failed. Retrying, 2 of 3.
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
gecko_client/admin_api.go:60:59: not enough arguments in call to api.rpcRequester.makeRpcRequest
	have (string, string)
	want (string, string, map[string]interface {})
The command "eval go get -v -t ./... " failed. Retrying, 3 of 3.
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
gecko_client/admin_api.go:60:59: not enough arguments in call to api.rpcRequester.makeRpcRequest
	have (string, string)
	want (string, string, map[string]interface {})

File: gecko_client/admin_api.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ type GetNodeIDResponse struct {
 }
 
 func (api AdminApi) GetPeers() ([]Peer, error) {
-	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(adminEndpoint, "admin.peers")
+	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(adminEndpoint, "admin.peers", make(map[string]interface{}))
```
Comment (galenmarchetti) [inline gecko_client/admin_api.go:46]: you'll need to do this at line 60 in this file as well, in the GetNodeID() call

travisCI currently failing with

The command "eval go get -v -t ./... " failed. Retrying, 2 of 3.
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
gecko_client/admin_api.go:60:59: not enough arguments in call to api.rpcRequester.makeRpcRequest
	have (string, string)
	want (string, string, map[string]interface {})
The command "eval go get -v -t ./... " failed. Retrying, 3 of 3.
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
gecko_client/admin_api.go:60:59: not enough arguments in call to api.rpcRequester.makeRpcRequest
	have (string, string)
	want (string, string, map[string]interface {})

Comment (mieubrisse) [reply gecko_client/admin_api.go:46]: fixt

**Pull request merged**